search-app/src/cmr/search/services/query_service.clj

```diff
@@ -355,9 +357,8 @@
                                   #(apply max (map :revision-id %))
                                   (group-by :concept-id (:items results)))
           highest-revisions (filter
-                             (fn [coll]
-                               ((set highest-coll-revisions)
-                                [(:concept-id coll) (:revision-id coll)]))
+                             #((set highest-coll-revisions)
```

Comment:
@chris-durbin give me an action to change this from this version to the code on master. Now you are changing it back. :)